test/metabase/pulse/render/body_test.clj

```diff
@@ -439,15 +439,15 @@
 (defn- render-multiseries-area-graph [results]
   (body/render :area :inline pacific-tz render.tu/test-combo-card nil results))
 
-(deftest render-area-graph-tet
+(deftest render-area-graph-test
   (testing "Render an area graph with non-nil values for the x and y axis"
     (is (has-inline-image?
           (render-area-graph {:cols default-columns
                               :rows [[10.0 1] [5.0 10] [2.50 20] [1.25 30]]}))))
   (testing "Render a stacked area graph"
     (is (has-inline-image?
-          (render-stack-area-graph {:cols default-columns
-                                    :rows [[10.0 1] [5.0 10] [2.50 20] [1.25 30]]}))))
+          (render-stack-area-graph {:cols default-multi-columns
+                                    :rows [[10.0 1 1231 1] [5.0 10 nil 111] [2.50 20 11 1] [1.25 nil 1231 11]]}))))
```

Comment:
The reason rows are changed is that the data was mocked incorrectly.

If you look at `render-stack-area-graph`

https://github.com/metabase/metabase/blob/a5c6a8b5ce09fbc8c455bec2fda1616ea01ed07e/test/metabase/pulse/render/body_test.clj#L436-L437

https://github.com/metabase/metabase/blob/a5c6a8b5ce09fbc8c455bec2fda1616ea01ed07e/test/metabase/pulse/render/test_util.clj#L42-L51

you'll see that the card used is basically the same as the combo card but have `:stackable.stack_type "stacked"`

That means the values for `:cols` and `:rows` need to look like the combo test too.

https://github.com/metabase/metabase/blob/a5c6a8b5ce09fbc8c455bec2fda1616ea01ed07e/test/metabase/pulse/render/body_test.clj#L465-L467